pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go

```diff
@@ -107,7 +121,7 @@ func TestSingleZone(t *testing.T) {
 			},
 		},
 		{
-			name: "label zone failure domain matched",
+			name: "label zone failure domain beta label matched",
```

Comment:
s/label zone failure domain beta label matched/beta zone label matched

ditto for the other test names below.